recipes/hla-asm/post-link.sh

```diff
@@ -0,0 +1,11 @@
+HLA_ASM_DIR=$PREFIX/opt/hla-asm
+REF_DIR=$HLA_ASM_DIR/reference_HLA_ASM 
+DL_CMD="cd $HLA_ASM_DIR && wget 'https://www.dropbox.com/s/mnkig0fhaym43m0/reference_HLA_ASM.tar.gz' && tar -xvzf reference_HLA_ASM.tar.gz"
+MESSAGE=""
+if [ ! -d $REF_DIR ]; then
+	echo $DL_CMD
+	{ eval $DL_CMD && MESSAGE="downloaded reference (~800MB)"; } || { MESSAGE="could not download reference. Please download it manually, like: $DL_CMD"; }
```

Comment:
Is is necessary to download the 800 MB reference automatically at installation time? Seems impolite if the user is not expecting it.

Reply:
yes, agreed. the user might wonder why the installation is not finishing. The program won't run without the 800MB reference. I thought it would be most convenient for the user to have it done automatically and would prefer to inform the user beforehand instead of afterwards but I do not know how. Do you think it is better to just print the message that the user must download it himself?

Reply:
I guess the most convenient thing would be to provide a downloader script (effectively copying what you have in the post-link) symlinked to the `bin/` dir. And print the message in post-link on how the user can do that. Might want to get the upstream authors involved as well...if they are recommending conda installation then the tool's documentation should also point out that the download step is required.

Reply:
Thanks. Need the `$` on `DL_CMD` (see [here](https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=18619&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=1585)).

Now that I look at it more closely, https://github.com/DiltheyLab/HLA-LA does have releases. It would be preferable to use the GitHub release tarball as as the source, and copy the relevant files over. That way the autobump bot will be able to detect new versions and create PRs automatically when it's updated.